nncf/torch/nncf_network.py

```diff
@@ -699,6 +698,9 @@ def get_node_to_op_address_mapping(self) -> Dict[NNCFNodeName, OperationAddress]
             retval[nncf_node.node_name] = op_address
         return retval
 
+    def set_compression_controller(self, ctrl: 'PTCompressionAlgorithmController'):
+        self.compression_controller = ctrl
```

Comment:
How about add short name for `compression_controller`, for example `model.nncf.ctrl` or  `model.nncf.controller`?`compression_controller` looks too long.

Reply:
I thought that it would be better to be more explicit here in the naming.